src/components/AvatarCropModal/Slider.js

```diff
@@ -36,29 +45,43 @@ function Slider(props) {
         transform: [{translateX: sliderValue.value}],
     }));
 
+    const panGesture = Gesture.Pan()
+        .minDistance(5)
+        .onBegin(() => {
+            runOnJS(setTooltipIsVisible)(false);
+            props.gestureCallbacks.onBegin();
+        })
+        .onChange((event) => {
+            props.gestureCallbacks.onChange(event);
+        })
+        .onFinalize(() => {
+            runOnJS(setTooltipIsVisible)(true);
+            props.gestureCallbacks.onFinalize();
+        });
+
     // We're preventing text selection with ControlSelection.blockElement to prevent safari
     // default behaviour of cursor - I-beam cursor on drag. See https://github.com/Expensify/App/issues/13688
     return (
         <View
             ref={ControlSelection.blockElement}
             style={styles.sliderBar}
         >
-            <PanGestureHandler
-                onBegan={() => setTooltipIsVisible(false)}
-                onEnded={() => setTooltipIsVisible(true)}
-                onGestureEvent={props.onGesture}
-            >
+            <GestureDetector gesture={panGesture}>
                 <Animated.View style={[styles.sliderKnob, rSliderStyle]}>
                     {tooltipIsVisible && (
                         <Tooltip
                             text={props.translate('common.zoom')}
                             shiftVertical={-2}
                         >
-                            <View style={[styles.sliderKnobTooltipView]} />
+                            {/* pointerEvents='none' is a workaround to make sure the pan gesture works correctly on mobile safari */}
+                            <View
+                                style={[styles.sliderKnobTooltipView]}
+                                pointerEvents="none"
```

Comment:
Isn't there some defined style value such as `PointerEventsNone` that should be used here instead of literal?

Reply:
Yes, there is